pkg/vectorize/date/date.go

```diff
@@ -22,3 +27,18 @@ func datetimeToDate(xs []types.Datetime, rs []types.Date) []types.Date {
 	}
 	return rs
 }
+
+func dateStringToDate(xs *types.Bytes, ns *nulls.Nulls, rs []types.Date) []types.Date {
+	for i := range xs.Lengths {
+		str := string(xs.Get(int64(i)))
+		d, e := types.ParseDatetime(str, 6)
+		if e != nil {
```

Comment:
You may need to look at pgsql or sqlserver.

Reply:
yeah, I will change to return err, not set null when string is err